source/loop/apis.rst

```diff
@@ -914,6 +914,57 @@ POST /rooms
             "roomUrl": "http://localhost:3000/static/#rooms/pPVoaqiH89M"
         }
 
+PATCH /rooms
+~~~~~~~~~~~~
+
+    **Requires owner authentication**
+
+    Remove given rooms
+
+    Request body parameters:
+
+    - **deleteRoomTokens**, a list of rooms to delete.
+
+    Response body parameters:
+
+    - **deletedRoomTokens**, a list of room's tokens actually deleted.
+
+    Potential HTTP error responses include:
+
+    - **404 Not Found:**  If none of the given roomTokens where found for this user.
+
+    Example::
+
+        echo '{"deleteRoomTokens": ["pPVoaqiH89M"]}' | http PATCH localhost:5000/v1/rooms -v \
```

Comment:
Not that I know of:

    $ http -v POST http://localhost/test toto='["titi", "tutu"]'
    {
      "toto": "[\"titi\", \"tutu\"]"
    }

    $ http -v POST http://localhost/test toto=titi,tutu
    {
        "toto": "titi,tutu"
    }